### PR TITLE
Minikube on ubuntu

### PR DESCRIPTION
**Pull request by romanprog**
The problem that was last time did not reproduce. (https://github.com/shalb/cluster.dev/pull/10#issuecomment-593950979)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -70,5 +70,11 @@
       "wordpress",
       "workdir",
       "yaml's"
-    ]
+    ],
+    "cSpell.ignoreWords": [
+      "file"
+    ],
```
Comment (MaxymVlasov) [inline .vscode/settings.json:76]: No idea why you ignore this word - cSpell know `file`
```suggestion
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -70,5 +70,11 @@
...
+    "files.autoSave": "onFocusChange",
+    "files.autoSaveDelay": 100,
+    "editor.fontSize": 13
```
Comment (MaxymVlasov) [inline .vscode/settings.json:79]: Looks like you accidentally add your global settings to local repo.
Please, remove it
```suggestion
    ]
```

File: terraform/aws/minikube/k8s-userdata.tpl.sh [hunk 3]
```diff
@@ -2,16 +2,17 @@
...
 # Install AWS utils
-yum install -y wget unzip python3-pip.noarch
+apt-get update
+apt-get install -y wget unzip python3-pip
 pip3 install awscli
 export PATH=\$$PATH:/usr/local/bin
 
 # Copy kubeconfig to S3 bucket
-aws s3 cp /home/centos/kubeconfig s3://${cluster_name}/kubeconfig_${cluster_name}
+aws s3 cp /home/ubuntu/kubeconfig s3://${cluster_name}/kubeconfig_${cluster_name}
```
Comment (MaxymVlasov) [inline terraform/aws/minikube/k8s-userdata.tpl.sh:11]: Maybe good idea will be use `$HOME`?

Like this: `"$HOME/kubeconfig"`

Comment (voatsap) [reply terraform/aws/minikube/k8s-userdata.tpl.sh:11]: keep to next time this change

File: bin/aws_minikube.sh [hunk 4]
```diff
@@ -81,7 +81,6 @@ function aws::minikube::deploy_cluster {
                 -backend-config='key=$cluster_name/terraform.state' \
                 -backend-config='region=$cluster_cloud_region'"
 
-    # TODO: Minikube module is using Centos7 image which requires to be accepted and subscribed in MarketPlace: https://github.com/shalb/cluster.dev/issues/9
     # To do so please visit https://aws.amazon.com/marketplace/pp?sku=aw0evgkw8e5c1q413zgy5pjce
```
Comment (voatsap) [inline bin/aws_minikube.sh:84]:
```suggestion
    
```

File: bin/aws_minikube.sh [hunk 5]
```diff
@@ -128,7 +127,6 @@ function aws::minikube::destroy_cluster {
...
```
Comment (voatsap) [inline bin/aws_minikube.sh:130]:
```suggestion
```

File: bin/aws_minikube.sh (hunk 4 above)
Comment (voatsap) [inline bin/aws_minikube.sh:84]:
```suggestion
```

**Pull request merged**